player-service/src/main/java/org/libertybikes/player/service/PlayerService.java

```diff
@@ -70,6 +71,14 @@ public String createPlayer(@QueryParam("name") String name, @QueryParam("id") St
         return p.id;
     }
 
+    @POST
+    @Produces(MediaType.TEXT_HTML)
+    @Path("/bot")
+    public String createBot(@QueryParam("name") String name) {
+        System.out.println("registering bot: " + name);
+        return createPlayer(name, generateKey());
+    }
+
```

Comment:
Instead of adding a new endpoint, can we just re-use the existing `createPlayer()` method/endpoint and have the caller (frontend) perform the ID generation?